helm-chart/splunk-kubernetes-logging/templates/psp.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.podSecurityPolicy.create -}}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "splunk-kubernetes-logging.fullname" . }}
+  labels:
+    app: {{ template "splunk-kubernetes-logging.name" . }}
+    chart: {{ template "splunk-kubernetes-logging.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+    apparmor.security.beta.kubernetes.io/allowedProfileNames: 'runtime/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+    apparmor.security.beta.kubernetes.io/defaultProfileName:  'runtime/default'
+spec:
+  privileged: true
```

Comment:
@matthewmodestino is this really required? this would violate other policies. Is it possible to run with `privileged: false`?

Reply:
I do believe it is required for the logging pod, (at least it is in Openshift, which is where this looks to have been inherited from) as it has to be able to read the json driver files from disk, which are owned by root.

Reply:
doubling back on this, i have been able to run as `priviledge: false` as long as context allows `runAsUser: 0`